Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'bzip2-ruby', :github => 'chewi/bzip2-ruby', :branch => 'master'
```

Comment:
@isimluk - thanks for the input. Would it sound reasonable that this api will fetch the bzipped xml file?